### PR TITLE
Add Coinsquare to list of exchanges

### PR DESCRIPTION
**Pull request by jaydeverett**
Add Coinsquare exchange to bitcoincash.org list of exchanges.

Coinsquare is Canada's leading digital currency exchange. Using our platform, Canadians can buy, sell, fund and withdraw BCH.

### CODE REVIEW

File: _includes/home/exchanges.html [hunk 1]
```diff
@@ -133,9 +133,9 @@ <h2>{% t 'headings.exchanges' %}</h2>
             <div class="col-sm-4 col-md-3 vertical-align to-animate">
                 <a href="https://exmo.com" target="_blank"><img alt="Exmo" src="/img/exchanges/exmo.png"></a>
             </div>
-            <div class="col-sm-12 text-center">
-                <a href="/get-listed.html" class="btn btn-default">{% t 'headings.listingrequest' %}</a>
+            <div class="col-sm-4 col-md-3 vertical-align to-animate">
```
Comment (maplesyrupsucker) [inline _includes/home/exchanges.html]: Can you revise to leave the Listing Request button in place as the final col-sm-12?

**Comment by jaydeverett** [comment]
@maplesyrupsucker, made the fix

**Comment by maplesyrupsucker** [comment]
@jaydeverett we're making some changes to acceptance criteria. We're going to be focusing on those exchanges that focus on Bitcoin Cash or at the very least promote it prominently. 

If possible can you consider adding Bitcoin Cash to the secondary heading line on your website to make it more clear that Bitcoin Cash is traded and accepted on your exchange.

image

**Comment by markblundeberg** [comment]
Worth noting that Coinsquare currently has BCH listed as "BAB". :joy:

**Comment by maplesyrupsucker** [comment]
image

Closing this until @jaydeverett / CoinSquare cut the bull.

Thanks for the heads up @markblundeberg

**Pull request closed without merging**